package.json

```diff
@@ -24,7 +24,7 @@
     "stylelint": "^11.1.1"
```

Comment:
Can you also update the `stylelint` dependency here to latest?

Reply:
Done!